backend/src/components/secureExchange.js

```diff
@@ -625,8 +625,13 @@ async function getEdxUsers(req, res) {
   const token = getAccessToken(req);
   validateAccessToken(token, res);
 
-  checkEDXUserSchoolAdminPermission(req, res);
-  checkEDXUserAccess(req, res, 'SCHOOL', req.query.schoolID);
+  if(req.query.districtID){
+    checkEDXUserDistrictAdminPermission(req, res);
+    checkEDXUserAccess(req, res, 'DISTRICT', req.query.districtID);
+  }else{
+    checkEDXUserSchoolAdminPermission(req, res);
+    checkEDXUserAccess(req, res, 'SCHOOL', req.query.schoolID);
+  }
```

Comment:
minor nitpik. Seems like sometimes we check for School first and then else = district. (like on line 915). Should we keep this consistent that we check for school first? and else = district. 
. 
Or we can do else if (req.query.districtID) to be more specific.